modules/web-element.mjs

```diff
@@ -340,6 +340,20 @@ class FontFamily extends WebElementFunction{
         return s
     }
 }
+class checkColorGreen extends WebElementFunction{
```

Comment:
Please add JSDoc comments to describe this class.

Reply:
okay